pset-iterators.js

```diff
@@ -183,9 +215,18 @@ Expected Result Shape: An array of objects (with each object representing a pres
 console.log('Problem 7:')
 
 // Add your code below this line
+const shortTermPresidents = presidents.filter(function (shortPresidentialTerms) {
+  const tookOffice = parseInt(shortPresidentialTerms.took_office)
+  const leftOffice = parseInt(shortPresidentialTerms.left_office)
 
+  if (leftOffice - tookOffice < 4) {
+    return shortPresidentialTerms
+  }
+})
+
+console.log(shortTermPresidents)
```

Comment:
@OcielC Good job here using the filter method to return all of the presidents who served for 4 years or less. Nice job using the parseInt on the `took_office` `left_office` as that method returns only the first numbers in a string.

Take a look at this solution [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L201)